### PR TITLE
Benkuhn/build instructions update

### PR DESCRIPTION
**Pull request by BenJKuhn**
This adds a note about required SDK version to the source build instructions.

### CODE REVIEW

**Comment by BenJKuhn** [comment]
I'll put both in. 1809 is the official name on the download page.

________________________________

File: src/readme.md [hunk 1]
```diff
@@ -9,13 +9,14 @@ xlang uses CMake instead of vcxproj in order to support cross platform builds.
...
 Building xlang for Windows requires the following to be installed:
-
 * [Visual Studio 2017](https://developer.microsoft.com/windows/downloads), version 15.8.7 or later.
   * Visual Studio's Desktop Development with C++ workload installation is required.
 * [CMake](https://cmake.org/), version 3.9 or later
   * Visual Studio's Desktop Development with C++ workload includes CMake 3.11
 * [Ninja](https://ninja-build.org/), version 1.8 or later
   * Visual Studio's Desktop Development with C++ workload includes Ninja 1.8.2
+* [Windows 10 SDK](https://developer.microsoft.com/en-US/windows/downloads/windows-10-sdk), Version 1809 or later
+  * Visual Studio does not install this by defualt with  version 15.8.7 or 15.8.8. It is an optional component in Visual Studio 15.8.
```
Comment (Scottj1s) [inline src/readme.md]: >defualt [](start = 43, length = 7)

typo

Comment (Scottj1s) [inline src/readme.md]: >  [](start = 56, length = 1)

space

File: src/readme.md [hunk 2]
```diff
@@ -9,13 +9,14 @@ xlang uses CMake instead of vcxproj in order to support cross platform builds.
 ### Prerequisites
 
 Building xlang for Windows requires the following to be installed:
-
 * [Visual Studio 2017](https://developer.microsoft.com/windows/downloads), version 15.8.7 or later.
   * Visual Studio's Desktop Development with C++ workload installation is required.
```
Comment (Scottj1s) [inline src/readme.md]: >C++ workload [](start = 45, length = 12)

"and Visual Studio SDK" (for natvis project)

**Pull request merged**